shared/productTypes.ts

```diff
@@ -591,6 +595,59 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 			explicitSingleDayOfWeek: 'Friday',
 		},
 	},
+	tierthree: {
+		productTitle: () => 'Digital + Print',
+		friendlyName: () => 'digital + print subscription',
+		shortFriendlyName: 'Print + Digital',
+		productType: 'tierthree',
+		groupedProductType: 'recurringSupport',
+		allProductsProductTypeFilterString: 'Weekly',
+		urlPart: 'digital+print',
+		softOptInIDs: [
+			SoftOptInIDs.SupportOnboarding,
+			SoftOptInIDs.SupporterNewsletter,
+			SoftOptInIDs.DigitalSubscriberPreview,
+			SoftOptInIDs.GuardianWeeklyNewsletter,
+		],
+		getOphanProductType: () => 'PRINT_SUBSCRIPTION', //TODO what should this be?
+		renewalMetadata: {
+			//TODO Don't think we need this, all T3s are auto renewing
```

Comment:
correct, there were some really old migrated in GWs that were renewed manually every year (or up to 3 years in some cases)

Reply:
I've removed that config